test/test.bats

```diff
@@ -88,15 +90,18 @@ load test_helper
 
   [ "$status" -eq 0 ]
 
-  LDAP_CID=$(docker run -h ldap.example.org -e LDAP_TLS=false --volume $PWD/VOLUMES/database:/var/lib/ldap --volume $PWD/VOLUMES/config:/etc/ldap/slapd.d -d $NAME:$VERSION)
+  # TODO: KEEP_EXISTING_CONFIG=true seems required to pass the test.
```

Comment:
@BertrandGouny I haz no idea what I'm doing here 🤷‍♂️ 
Does it make sense to add `KEEP_EXISTING_CONFIG=true` here, or not?